mlflow/utils/_capture_modules.py

```diff
@@ -120,7 +122,10 @@ def _load_pyfunc_patch(*args, **kwargs):
             with cap_cm:
                 model = original(*args, **kwargs)
                 if input_example is not None:
-                    model.predict(input_example)
+                    if "params" in inspect.signature(model.predict).parameters:
+                        model.predict(input_example, params=params)
+                    else:
+                        model.predict(input_example)
```

Comment:
Is there a pyfunc wrapper whose `predict` method doesn't have `params`?

Reply:
Ah it's pyfunc, then no :) We could remove this. Good catch!!